bc_obps/registration/api/user_operator.py

```diff
@@ -97,7 +98,8 @@ def get_user_operator_operator(request):
 
 
 @router.get("/user-operator-id", response={200: UserOperatorIdOut, codes_4xx: Message}, url_name="get_user_operator_id")
-@authorize(["industry_user"], UserOperator.get_all_industry_user_operator_roles())
+# brianna I think it's fine for them to see this
```

Comment:
thoughts

Reply:
also agree

Reply:
Where do we use this endpoint? :D

Reply:
It used to be in the `select-operator` page but it must have been cut at some point